docs/applications/drupal/automatic-updates.md

```diff
@@ -23,7 +23,7 @@ The following update methods been disabled by Lagoon.
 ## Drupal Automatic Updates
 
 The [Automatic Updates](https://www.drupal.org/project/automatic_updates)
-contrib module is disabled by and it will also be disabled when it moves
+contrib module is disabled by Lagoon. The module will also be disabled when it moves
```

Comment:
I think I meant to say "disabled by default." We will use an environment variable to disable, so users would be able to override that if they want (for example, using/testing automatic updates locally).

Reply:
We should probably explain here how it is disable - what the variable is, and where it is set